plugins/processor/desensitize/processor_desensitize_test.go

```diff
@@ -34,84 +33,98 @@ func newProcessor() (*ProcessorDesensitize, error) {
 		RegexBegin:    "'password':'",
 		RegexContent:  "[^']*",
 	}
-	err := processor.Init(ctx)
-	return processor, err
+	return processor
+}
+
+func TestChineseSample(t *testing.T) {
```

Comment:
This test is useless. It can be passed on the original code.

Reply:
done